### PR TITLE
kRO & jRO Costume Update.

### PR DESCRIPTION
**Pull request by Jedzkie**
Added 8 Official kRO & jRO Costumes supported up to 2015-05-13aRagexe.
Change the AegisName of Music Decoration to its official name.
Update Status Change ID's. (Thanks to Rytech)

Note:
 • Some of the costumes items are missing or not implemented on this commit, but this is their official SC's.
 • SC's that don't have costume items:
   - SC_FLOWERSMOKE
   - SC_HAPPINESS_STAR
   - SC_MAGICAL_FEATHER

### CODE REVIEW

File: src/map/status.c [hunk 1]
```diff
@@ -1080,13 +1090,20 @@ void initChangeTables(void) {
...
 	status->dbs->ChangeFlagTable[SC_FENRIR_CARD] |= SCB_MATK | SCB_ALL;
 	
-	// Costume
+	// Costumes
 	status->dbs->ChangeFlagTable[SC_MOONSTAR] |= SCB_NONE;
 	status->dbs->ChangeFlagTable[SC_SUPER_STAR] |= SCB_NONE;
 	status->dbs->ChangeFlagTable[SC_STRANGELIGHTS] |= SCB_NONE;
 	status->dbs->ChangeFlagTable[SC_DECORATION_OF_MUSIC] |= SCB_NONE;
 	status->dbs->ChangeFlagTable[SC_LJOSALFAR] |= SCB_NONE;
-	status->dbs->ChangeFlagTable[SC_MERMAID_LONGING] |= SCB_NONE;
```
Comment (kyeme) [inline src/map/status.c]: Typo?

**Comment by Jedzkie** [comment]
Geez another failure....... Thanks bro.

**Comment by dastgir** [comment]
Where you got SC list?
This is going to need some thing, please don't merge this, until I check something in kRO

**Comment by Jedzkie** [comment]
@dastgir from Rytech sensei.

**Comment by dastgir** [comment]
Yes, but I have some info, which I need to verify

**Comment by dastgir** [comment]
correct the StatusID in const.txt, and it will be merged.

**Pull request merged**